src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto;
+
+import java.util.List;
+
+public class Lotto {
+
+	private static final int REQUIRED_COUNT = 6;
+
+	private final List<Number> numbers;
+
+	public Lotto(List<Number> numbers) {
+		validateNumbers(numbers);
+
+		this.numbers = numbers;
+	}
+
+	public Winner winningCheck(Lotto winLotto) {
+		return Winner.valueOf(
+			(int) numbers.stream()
```

Comment:
```suggestion
			numbers.stream()
```
형번환은 Winner에서 해줘도 되지 않을까요?